components/builder-web/app/origin/origin-page/origin-job-detail/origin-job-detail.component.html

```diff
@@ -0,0 +1,91 @@
+      <a (click)="backToGroups()">
+        Build Job Groups
+      </a>&nbsp;/ Group {{ this.id }}
+    </h3>
+    <section class="jobgroup-overview">
+      <table class="jobgroup-table">
+        <thead>
+          <tr>
+            <th>Initiated By</th>
+            <th>Group Id</th>
+            <th>Created</th>
+            <th>Updated</th>
+            <th>Build Time</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>{{ thisGroup.project_name }}</td>
+            <td>{{ thisGroup.id }}</td>
+            <td>{{ dateFor(thisGroup.created_at) }}</td>
+            <td></td>
+            <td></td>
+          </tr>
+        </tbody>
+      </table>
+      <hr>
+      <table class="jobgroup-table">
+        <thead>
+          <tr>
+            <th>Complete / Total</th>
+            <th colspan=5>Job Status</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>{{ completedCount }} / {{ totalCount }}</td>
+            <td>
+              <hab-icon [symbol]="iconFor('complete')" class="status complete" title="complete"></hab-icon>
+              {{ projectStateCount('success') }}
+            </td>
+            <td>
+              <hab-icon [symbol]="iconFor('failure')" class="status failure" title="failure"></hab-icon>
+              {{ projectStateCount('failure') }}
+            </td>
+            <td>
+              <hab-icon [symbol]="iconFor('pending')" class="status pending" title="pending"></hab-icon>
+              {{ projectStateCount('pending') }}
+            </td>
+            <td>
+              <hab-icon [symbol]="iconFor('processing')" class="status processing" title="processing"></hab-icon>
+              {{ projectStateCount('processing') }}
+            </td>
+            <td>
+              <hab-icon [symbol]="iconFor('cancelled')" class="status cancelled" title="cancelled"></hab-icon>
+              {{ projectStateCount('cancelled') }}
+            </td>
+          </tr>
+        </tbody>
+      </table>
+    </section>
+    <section class="jobgroup-list">
+      <ol class="nav-list">
+        <li class="heading">
+          <h4>Package</h4>
+          <h4>Job ID</h4>
+          <h4>Job Status</h4>
+          <h4>Updated</h4>
+        </li>
+        <li class="item" *ngFor="let project of projects" (click)="onSelectJob(project.name, project.job_id)">
+          <a>
+            <span class="column">
+              {{ project.name }}
+            </span>
+            <span class="column">
+              {{ project.job_id }}
+            </span>
+            <span class="column icons">
+              <hab-icon [symbol]="iconFor(project.state)" class="status {{ project.state | lowercase }}" title="{{ project.state | titlecase }}"></hab-icon>
+              <hab-icon symbol="chevron-right"></hab-icon>
+            </span>
+            <span class="column">
+            </span>
+          </a>
+        </li>
+      </ol>
+    </section>
+  </div>
+</div>
```

Comment:
it's good practice to put a newline at the end of every file. You should be able to tweak your editor to apply  one on every file-save. Be good to fix up the few of them in this PR.